content/applications/websites/ecommerce/maximizing_revenue/cross_selling.rst

```diff
@@ -1,55 +1,64 @@
+maximize the value of each one of your customers.
 
-Accessory products when checking out
-====================================
+It can be done via two different features:
 
-Accessories (e.g. for computers: mouse, keyboard) show up when the customer
-reviews the cart before paying.
+    - Accessory products on the checkout page.
+    - Optional products on a new *Add to Cart* screen.
 
-.. image:: ./media/cross_sell_accessory_web.png
-   :align: center
+.. _cross_selling/accessory:
 
-Select accessories in the *Sales* tab of the product detail page.
+Accessory Products
+==================
 
-.. image:: ./media/cross_sell_accessory.png
-   :align: center
+*Accessory Products* are products that are displayed when customers review their cart before paying.
+As an example, you could suggest a mouse or a keyboard when someone buys a computer.
 
-.. note::
-    There is an algorithm to figure out the best accessories to display in case
-    several items are added to cart. If any item is the accessory of several products
-    added to cart, it is most likely that it will be atop the list of suggested accessories.
+To add *Accessory Products*, go the :menuselection:`Website --> Products --> Products` and, under
+the **Sales Tab**, add your accessories to the **Accessory Products** field.
 
-Optional products when adding to cart
-=====================================
+    .. image:: cross_selling/accessory-product.png
+       :align: center
+       :alt: Add an accessory product to product form.
 
-Optional products are directly related to the item added to cart (e.g. for computers:
-warranty, OS software, extra components). Whenever the main product is added to cart,
-such a new screen pops up as an extra step.
+Now, go to your website and add the product to your cart. On the payment step, you can see your
+accessory products suggested to the customer.
 
-.. image:: ./media/cross_sell_optional_web.png
-   :align: center
+    .. image:: cross_selling/accessory-products-customer-view.gif
+       :align: center
+       :alt: See of payment view and accessory product suggestion.
 
-To publish optional products:
+.. _cross_selling/optional:
 
-* Install *eCommerce Optional Products* addon in *Apps* menu. Remove the default filter to
-  search on addons as well, otherwise only main apps show up.
+Optional Products
+=================
 
-.. image:: ./media/cross_sell_optional_install.png
-   :align: center
+*Optional products* are products directly related to an item. Each time the product is added to a
+cart, a pop-up appears, suggesting other products to buy.
 
-* Select optional items from the *Sales* tab of the product detail form.
+To enable optional products, go to :menuselection:`Website --> Configuration --> Settings` and tick
+*Optional Products*. Then, go to :menuselection:`Website --> Products --> Products` and open the
+item you want to add an optional product to. In the **Sales tab** of that product, add as many
+optional products as you want.
```

Comment:
I know "item" is best to avoid repeating the same words, but since we're talking about views in Odoo, I'm wondering if it wouldn't be best to write "product".